query/common/src/main/java/org/codice/ddf/admin/common/fields/base/scalar/IntegerField.java

```diff
@@ -17,10 +17,10 @@
 
 public class IntegerField extends BaseScalarField<Integer> {
 
-  public static final String DEFAULT_FIELD_NAME = "integer";
+  public static final String INTEGER_FIELD_DEFAULT_FIELD_NAME = "integer";
```

Comment:
maybe `DEFAULT_INTEGER_FIELD_NAME` instead? That convention might work for most of these base classes.

Reply:
That sounds better 👍